README.md

```diff
@@ -334,6 +343,32 @@ using the formula `0.99 * 100 = 99`.
 > 💡 **Note:** : As you can see for both query combinations we end up with the same calculated
 > value for the service level indicator.
 
+The optional `metadata` key can be used to pass extraneous data to the data source,
+for example, if your data source accepts variables, they could be passed via the
+`metadata`.
+
+An example is an internal tool which uses a templating feature to ease the maintenance
+of SLOs by not repeating the same queries were there are only small differences.
+The internal tool can then generate the final OpensLO specification based on the input described.
+
+```yaml
+ratioMetric:
+    counter: true
+    good:
+        source: prometheus
+        queryType: query
+        query: sum:requests.duration{*}
+        metadata:
+        bucket: "0.25"
```

Comment:
Should `bucket` and `exclude_errors` be indented to indicate sub fields of `metadata`?

Reply:
fixed in b1a158fa